src/commands/General/Chat Bot Info/stats.js

```diff
@@ -37,7 +37,7 @@ module.exports = class extends Command {
             .addField("❯ Total Commands Ran", this.client.settings.counter.total.toLocaleString(), true)
             .addField("❯ CPM", cpm, true)
             .addField("❯ Listeners", listeners, true)
-            .addField("❯ Shards", `${this.client.shard.id + 1} / ${this.client.shard.count}`, true)
+            .addField("❯ Sharding", `**Cluster:** ${Number(process.env.CLUSTER_ID) + 1} / ${process.env.CLUSTER_CLUSTER_COUNT} | **Shard:** ${this.client.shard.id + 1} / ${this.client.shard.shardCount}`, true) // eslint-disable-line no-process-env
```

Comment:
this.client.shard is modified to have this information instead of adding a comment to disable eslint